app/models/bill.rb

```diff
@@ -57,6 +60,15 @@ def update_cosponsors(cosponsors)
     end
   end
 
+  def timeline
+    running_count = 1 # sponsor
+    sponsorships.where.not(cosponsored_at: nil).order("DATE(cosponsored_at)")
+                .group("DATE(cosponsored_at)").count.map do |date, count|
```

Comment:
Use 2 (not 12) spaces for indenting an expression spanning multiple lines.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.